activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Disable `prepared_statements` for mysql2 adapter
+
+    Currently, `prepared_statements` defaults to `true` although this is only supposed to happen
```

Comment:
Assumed this behavior change has been made since https://github.com/rails/rails/pull/44591 , this behavior is not available to Rails 7.0.z. then "Currently, `prepared_statements` defaults to `true`" may cause misunderstandins. that "current" behavior as Rails 7.0.z or older verions of Rails.